### PR TITLE
Add eslint

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: src/schema/resolvers.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint no-underscore-dangle: [2, { "allow": ["_id"] }] */
```
Comment (frankobe) [inline src/schema/resolvers.js]: should we disable `no-underscore-dangle` in general?

File: src/sync/models/decentralizedOracle.js [hunk 2]
```diff
@@ -13,39 +13,37 @@ class DecentralizedOracle {
...
       address: this.contractAddress,
       txid: this.txid,
-      topicAddress:this.eventAddress,
+      topicAddress: this.eventAddress,
       status: 'VOTING',
       token: 'BOT',
       name: this.name,
       options: this.options,
-      optionIdxs: optionIdxs,
+      optionIdxs,
```
Comment (frankobe) [inline src/sync/models/decentralizedOracle.js]: find this syntax confusing

File: src/schema/resolvers.js [hunk 3]
```diff
@@ -112,63 +118,71 @@ function buildVoteFilters({ OR = [], oracleAddress, voterAddress, voterQAddress,
...
+        const resp = await fetch('https://testnet.qtum.org/insight-api/status?q=getInfo');
+        const json = await resp.json();
+        chainBlockNum = json.info.blocks;
+      } catch (err) {
         console.error(`Error GET https://testnet.qtum.org/insight-api/status?q=getInfo: ${err.message}`);
       }
 
-      return {'syncBlockNum': syncBlockNum, 'chainBlockNum': chainBlockNum }
-    }
+      return { syncBlockNum, chainBlockNum };
```
Comment (frankobe) [inline src/schema/resolvers.js]: find this syntax confusing

File: src/schema/resolvers.js [hunk 4]
```diff
@@ -112,63 +118,71 @@ function buildVoteFilters({ OR = [], oracleAddress, voterAddress, voterQAddress,
...
-      var cursor = Topics.cfind(query);
+    allTopics: async (root, {
+      filter, orderBy, limit, skip,
+    }, { db: { Topics } }) => {
+      const query = filter ? { $or: buildTopicFilters(filter) } : {};
+      let cursor = Topics.cfind(query);
       cursor = buildCursorOptions(cursor, orderBy, limit, skip);
 
-      return await cursor.exec();
+      return cursor.exec();
```
Comment (frankobe) [inline src/schema/resolvers.js]: why not `await` here

File: src/schema/resolvers.js (hunk 1 above)
Comment (dwalintukan) [reply src/schema/resolvers.js]: Ya if you want to. Although I think using underscore for naming convention is a bit weird looking.

File: src/schema/resolvers.js (hunk 4 above)
Comment (dwalintukan) [reply src/schema/resolvers.js]: async functions are already wrapped in Promise.resolve
https://eslint.org/docs/rules/no-return-await

File: src/schema/resolvers.js (hunk 3 above)
Comment (dwalintukan) [reply src/schema/resolvers.js]: it's called shortform in JS. basically if the key and value are the same name, you just need to use it once.

File: src/sync/models/decentralizedOracle.js (hunk 2 above)
Comment (dwalintukan) [reply src/sync/models/decentralizedOracle.js]: same thing, object-shorthand. its a new ES6 standard, but if you want it disabled we can:
https://eslint.org/docs/rules/object-shorthand

**Pull request merged**